src/mcts/node.cc

```diff
@@ -243,8 +243,12 @@ void Node::CancelScoreUpdate(int multivisit) {
 
 void Node::FinalizeScoreUpdate(float v, float d, int multivisit) {
   // Recompute Q.
+  float old_delta = (v - q_);
   q_ += multivisit * (v - q_) / (n_ + multivisit);
   d_ += multivisit * (d - d_) / (n_ + multivisit);
+  float new_delta = (v - q_);
+  // Need to divide by 4 to scale to 0..1 range probability.
+  q_squared_diff_ += 0.25f * old_delta * new_delta;
```

Comment:
This formula is incorrect, the increment has to be multiplied by `multivisit`. Compare this with the one used in #791, which is exact, even in the multivisit case.

Now the variance is underestimated, which might explain the rather low clopped values for ci_alpha.

Reply:
I didn't want to deal with multivisits yet. #791 is not correct either as it gives different answer depending on the order of the samples (Check for example [0, 1] vs [1, 0]).

`ci_alpha` is very similar to what I clopped in LZ and lack of multivisits probably doesn't affect it. The reason is that the evaluations aren't really normally distributed. See discussion in LZ PR.

Reply:
You are already dealing with multivisits (you cannot circumvent them), as you divide the variance by N-1 (which includes multivisits).

I insist #791 is correct and can give you the derivation if you're interested.

Reply:
You are correct about the multivisits and I should add support for it.

Check the case above and keep in mind that `n_` is initialized to zero and `q_` is mean of the previous samples, not mean of all samples. With multivisit = 1 it should match Welford's algorithm, but it doesn't.

Reply:
Okay, forget the previous messages. I checked it more carefully and it seems to give correct answer for multivisit = 1. I trust it works for other multivisit values too.

Reply:
Okay, here's the derivation anyway, as reference. 
![IMG_20190328_203618293](https://user-images.githubusercontent.com/38748663/55187536-97cab080-5199-11e9-8090-a182c7796278.jpg)